packages/common-ui/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "build": "tsup-node",
-    "lint": "eslint src/**/*.{ts,tsx}",
+    "lint": "eslint src/**/*.{ts,tsx} src/*.{ts,tsx}",
```

Comment:
The first one (`eslint src/**/*.{ts,tsx}`) should already match `src/*.ts` and `src/*.tsx` files

Reply:
exactly! however I came across: https://github.com/NorthernTechHQ/nt-gui/pull/148/commits/9cadf3f7c3a6192933c3074ccd7d706f5f63ab0b and eslint didn't see any issue - which made me think this parameter doesn't apply globbing as expected... 😕